tests/foreman/ui_airgun/test_sync.py

```diff
@@ -0,0 +1,177 @@
+"""Test class for Custom Sync UI
+
+:Requirement: Sync
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.constants import (
+    DISTRO_RHEL6, DISTRO_RHEL7,
+    FAKE_1_YUM_REPO,
+    FEDORA23_OSTREE_REPO,
+    REPOS,
+    REPOSET,
+    PRDS,
+)
+from robottelo.decorators import (
+    fixture,
+    run_in_one_thread,
+    skip_if_not_set,
+    tier2,
+    upgrade,
+)
+from robottelo.decorators.host import skip_if_os
+from robottelo.products import (
+    RepositoryCollection,
+    RHELCloudFormsTools,
+    SatelliteCapsuleRepository,
+)
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_custom_product(module_org):
+    return entities.Product(organization=module_org).create()
+
+
+@fixture(scope='module')
+def org_manifest(module_org):
+    manifests.upload_manifest_locked(module_org.id)
+
+
+@tier2
+def test_positive_sync_custom_repo(session, module_org, module_custom_product):
+    """Create Content Custom Sync with minimal input parameters
+
+    :id: 00fb0b04-0293-42c2-92fa-930c75acee89
+
+    :expectedresults: Sync procedure is successful
+
+    :CaseImportance: Critical
+    """
+    repo = entities.Repository(
+        url=FAKE_1_YUM_REPO, product=module_custom_product).create()
+    with session:
+        session.organization.select(org_name=module_org.name)
+        results = session.sync_status.synchronize([
+            (module_custom_product.name, repo.name)])
+        assert len(results) == 1
+        assert results[0] == 'Syncing Complete.'
+
+
+@run_in_one_thread
+@skip_if_not_set('fake_manifest')
```

Comment:
This check doesn't make much sense as fake manifest is used for module org, meaning all the tests are dependent on fake_manifest. Can we apply it to all the tests in the module somehow? Or maybe use 2 different fixtures (module_org & org_with_manifest)?

Reply:
@abalakh we have 2 tests that will run in parallel and use manifest 2 other tests run in thread and use manifest